src/abstracts/SablierV2ProxyTarget.sol

```diff
@@ -707,26 +691,14 @@ contract SablierV2ProxyTarget is
         }
     }
 
-    /// @dev Helper function to transfer funds from the proxy owner to the proxy using Permit2 and, if needed, approve
-    /// the Sablier contract to spend funds from the proxy.
+    /// @dev Helper function to transfer funds from the proxy owner to the proxy, if needed, approve the Sablier
+    /// contract to spend funds from the proxy.
```

Comment:
Needs to be rewritten to explain that this function is meant to be implemented by child contracts.